content/04.results.md

```diff
@@ -41,75 +41,23 @@ We plotted the 95% confidence interval for each reported token.
 We plotted the 95% confidence interval for each reported token.
 **E.** The tokens with the largest absolute differences in abundance appear to be stylistic.
-](https://raw.githubusercontent.com/danich1/annorxiver/77e9d6437069bafec7f0291b9a6c09ad50374b58/figure_generation/output/figure_one_panels.svg){#fig:corpora_comparison_panels width="100%"}
+](https://raw.githubusercontent.com/danich1/annorxiver/b8d20a1e7f59e43817cfdb8a5b0156b607b2c900/figure_generation/output/figure_one_panels.png){#fig:corpora_comparison_panels width="100%"}
 
 Documents within bioRxiv were slightly longer than those within PMCOA, but both were much longer than those from the control (NYTAC) (Table {@tbl:corpora_stats}).
 The average sentence length, the fraction of pronouns, and the use of the passive voice were all more similar between bioRxiv and PMC than they were to NYTAC(Table {@tbl:corpora_stats}).
 The Kullback–Leibler (KL) divergence of term frequency distributions between bioRxiv and PMCOA were low, especially among the top few hundred tokens (Figure {@fig:corpora_comparison_panels}A).
 As more tokens were incorporated, the KL divergence started to increase but remained much lower than the biomedical corpora compared against NYTAC.
 These findings support our notion that bioRxiv is linguistically similar to the PMCOA repository.
 
-Terms like "neurons", "genome", and "genetic", which are common in genomics and neuroscience, were more common in bioRxiv than PMCOA while others associated with clinical research, such as "clinical" "patients" and "treatment" were more common in PMCOA (Figure {@fig:corpora_comparison_panels}B and {@fig:corpora_comparison_panels}C).
-When controlling for the differences in the body of documents to identify textual changes associated with the publication process, we found that tokens such as "et" "al" were enriched for bioRxiv while "$\pm$", "–" were enriched for PMCOA (Figure {@fig:corpora_comparison_panels}D and {@fig:corpora_comparison_panels}E).
+Terms like "neurons", "genome", and "genetic", which are common in genomics and neuroscience, were more common in bioRxiv than PMCOA while others associated with clinical research, such as "clinical" "patients" and "treatment" were more common in PMCOA (Figure {@fig:corpora_comparison_panels}B, {@fig:corpora_comparison_panels}C and Supplementary Figure {@fig:corpora_comparison_panels_bio_v_pmid}).
+When controlling for the differences in the body of documents to identify textual changes associated with the publication process, we found that tokens such as "et" "al" were enriched for bioRxiv while "$\pm$", "–" were enriched for PMCOA (Figure {@fig:corpora_comparison_panels}D, {@fig:corpora_comparison_panels}E). 
+When removing special and single-character tokens, more figure-related terms appeared enriched for published articles, and more research-related articles appeared enriched for bioRxiv (Supplementary Figure {@fig:corpora_comparison_panels_pre_v_pub}).
```

Comment:
What is a "research-related article"? I am missing something. Do you mean terms associated with research vs terms associated with presentation (i.e., fig vs figure)? I am a bit confused here.

Reply:
>What is a "research-related article"? I am missing something. Do you mean terms associated with research vs terms associated with presentation (i.e., fig vs figure)? I am a bit confused here.

I meant to say terms related to research rather journal presentation. Will update this sentence to be more clear.